tests/ScannerTest.hack

```diff
@@ -83,8 +83,8 @@ final class ScannerTest extends BaseTest {
   }
 
   private function assertMapMatches(
-    array<string, string> $expected,
-    array<string, string> $actual,
+    KeyedContainer<string, string> $expected,
```

Comment:
Not dict?

Reply:
You might be on to something here.
I assumed this function compared the contents, not the type, since the body doesn't mention the type at all.
However, this was coded in a day before `varray<_>` and `darray<_, _>`.
So the author might have actually wanted to constrain the types to be the same too.
I'll dictify the typehints, just to be sure.

Reply:
Ah chucks, I can't do that on this branch, since `AutoloadMap\AutoloadMap` has darray keys. This change can be made in lexidor#1 though.